parser.py

```diff
@@ -1 +1,40 @@
-# TODO Xin - Add methods here that we will need. Try to think carefully about organization and how to separate functionality effectively.
\ No newline at end of file
+# TODO Xin - Add methods here that we will need. Try to think carefully about organization and how to separate functionality effectively.
+from bs4 import BeautifulSoup as bs
```

Comment:
Here's another important thing about aliases. It's sometimes okay to shorten module names, but BeautifulSoup is not a module. It's a class! bs4 is the module, and BeautifulSoup is a class within that module. Let's not alias those, because then its inconsistent what's aliased and what's not.